ui/imports/shared/controls/AccountSelector.qml

```diff
@@ -0,0 +1,144 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtQml 2.15
+
+import StatusQ 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1 as StatusQUtils
+
+import utils 1.0
+
+import shared.controls 1.0
+
+StatusComboBox {
+    id: root
+
+    enum Style {
+        Default,
+        Header
+    }
+
+    // input property for programatic selection
+    property string selectedAddress: ""
+    // output property for selected account
+    readonly property alias currentAccount: selectedEntry.item
+    readonly property string currentAccountAddress: root.control.currentValue ?? ""
```

Comment:
Could be a tradeoff between simple API and convenience 🤔 

Found it handy to have separate input and output properties because the input property doesn't need to break the binding internally. Having only one read/write property would probably result in multiple external `Bindings` objects `orSelectedAddressChanged` handlers on to handle the external address change. Would probably be harder to cover all corner cases when the `AccountSelector` is used.

Reply:
The problem here is probably a bit deeper, in `StatusComboBox` which is not taking advantage of `textRole` and `valueRole` properties. But the current solution is decent, I'm ok with leaving it as it is.